### ISSUE TITLE
invalid multibyte char (US-ASCII)

### ISSUE THREAD

**Issue by tralles** [barttenbrinke/munin-plugins-rails#16]
i have no idea why, but munin_passenger_queue is throwing errors:

./munin_passenger_queue:40: invalid multibyte char (US-ASCII)
./munin_passenger_queue:2: syntax error, unexpected $end, expecting tSTRING_CONTENT or tSTRING_DBEG or tSTRING_DVAR or tSTRING_END

**Comment by barttenbrinke** [comment]
What version of Ruby are you running?

**Comment by tralles** [comment]
# ruby -v

ruby 1.9.3p0 (2011-10-30 revision 33570) [x86_64-linux]

it's a ubuntu 12.04 installation …

**Comment by barttenbrinke** [comment]
Thats probably the issue.. let me see if I can reproduce this.

**Comment by tralles** [comment]
i found, it's the ë in „Michaël” - if i rename him to „Michael” the execution runs without errors …

**Comment by barttenbrinke** [comment]
Thats a workaround :) munin-plugins-rails should be able to work with UTF-8 chars in Ruby1.9.3 without any problems.
BTW: Please note that the patchlevel your running is not a recommended one.

**Comment by barttenbrinke** [comment]
I've pushed a new passenger_queue to master. Could you check if that solves the issue for you?

**Comment by tralles** [comment]
i'm sorry. but. aehm. how can i install a dedicated branch via gem install at the shell …

**Comment by barttenbrinke** [comment]
```
git clone https://github.com/barttenbrinke/munin-plugins-rails.git
cd munin-plugins-rails
gem install jeweler
rake build
gem install pkg/munin-plugins-rails-0.2.12.gem 
```

**Comment by tralles** [comment]
it seems to be one of those days:

error: The requested URL returned error: 403 while accessing https://github.com/barttenbrinke/munin-plugins-rails.git/info/refs

**Comment by barttenbrinke** [comment]
And if you try:

```
git clone git://github.com/barttenbrinke/munin-plugins-rails.git
```

**Comment by tralles** [comment]
many thanks for the advice - to install it from the master-branche worked now and _tada_ the utf-8 problem is solved also :)

**Comment by tralles** [comment]
but the log shows now:

2012/11/13-16:40:04 [29563] Error output from TN-production_munin_rails_database_time:
2012/11/13-16:40:04 [29563]     Request-log-analyzer, by Willem van Bergen and Bart ten Brinke - version 1.12.4
2012/11/13-16:40:04 [29563]     Website: http://railsdoctors.com

trying to execute TN-production_munin_rails_database_time shows me now:
Filepath unspecified. Exiting

do i have to remove and generate those sensors again?

**Comment by innovia** [comment]
please compile the gem and make it available, i only needed to add the encoding tuf-8 to it

**Comment by barttenbrinke** [comment]
Oh man, I'm really sorry. I thought I pushed this out 3 months ago :X Released 0.2.12 right away!

**Comment by innovia** [comment]
Thanks a lot

Ami mahloof

On 10 בפבר 2013, at 22:01, Bart ten Brinke notifications@github.com wrote:

Oh man, I'm really sorry. I thought I pushed this out 3 months ago :X
Released 0.2.12 right away!

—
Reply to this email directly or view it on
GitHubhttps://github.com/barttenbrinke/munin-plugins-rails/issues/16#issuecomment-13359863..
